community/io/src/test/java/org/neo4j/graphdb/mockfs/EphemeralFileSystemAbstraction.java

```diff
@@ -592,6 +613,17 @@ public void truncate( File file, long size ) throws IOException
         data.truncate( size );
     }
 
+    @Override
+    public long lastModifiedTime( File file ) throws IOException
+    {
+        EphemeralFileData data = files.get( canonicalFile( file ) );
+        if ( data == null )
+        {
+            throw new FileNotFoundException( "File " + file + " not found" );
```

Comment:
Could be, but all the other methods in `EphemeralFileSystemAbstraction` return FileNotFoundException when the file data is null. This is only following the pattern.